app/serializers/npq_participant_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+class NPQParticipantSerializer
+  include JSONAPI::Serializer
+  include JSONAPI::Serializer::Instrumentation
+
+  set_id :id
+  set_type :'npq-participant'
+
+  attributes :participant_id, :date_of_birth, :teacher_reference_number, :teacher_reference_number_verified, :active_alert,
```

Comment:
I don't believe `date_of_birth` is currently exposed. The provider doesn't need to know it

Reply:
should this endpoint return the course that is being undertaken? as i don't see that here.

Reply:
```date_of_birth``` is a column in the npq_profiles table, same as ```npq_course_id```

Reply:
should definitely be passing back the course_identifier rather than the npq_course_id. It should closely correspond to what they have to declare.

Reply:
If only it was that simple 😂 
I think really we will need to pass on a list of course identifiers, because a single participant can be doing more than one course.

Reply:
Only left few attributes for now as we can add more later i.e. ```email```, ```full_name```, ```npq_courses```, ```teacher_reference_number```, ```teacher_reference_number_validated```